build/release/common.sh

```diff
@@ -28,17 +28,27 @@ layout_root() {
     cp /etc/ssl/certs/ca-certificates.crt $dir/root/etc/ssl/certs
 }
 
+check_release_version() {
+    if echo ${RELEASE_VERSION} | grep -q -E '^([[:digit:]]+)\.([[:digit:]]+)\.([[:digit:]]+)$'; then
+        return 0
+    else
+        return 1
+    fi
+}
+
 github_get_release_id() {
     curl -4 -s -H "Authorization: token ${GITHUB_TOKEN}" \
        "https://api.github.com/repos/${GITHUB_USER}/${GITHUB_REPO}/releases/tags/${RELEASE_VERSION}" | jq -r '.id'
 }
 
-github_check_release() {
+github_create_release() {
     id=$(github_get_release_id)
 
     if [[ ${id} == "null" ]]; then
-        echo "ERROR: github release tag ${RELEASE_VERSION} was not found. Did you create the release?" 1>&2
-        return 1;
+        curl -4 -s -H "Authorization: token ${GITHUB_TOKEN}" \
```

Comment:
is it expected for a human to come by later on and fill in the release notes with content?